NEWS.md

```diff
@@ -55,6 +55,7 @@ also addresses the issue in the downstream functions `derive_vars_dt()` and `der
 
 - `derive_var_lstalvdt()` has been deprecated in favor of `derive_var_extreme_dt()` (#753).
 - `derive_vars_disposition_reason()` now is updated such that the default is populating `DCSREASP` only when `DSDECOD` is equal to `'OTHER'`, which is consistent with ADaMIG_v1.3 (#886).
+- `derive_vars_suppqual` has been removed from {admiral} as adding supplementary qualifiers is now done in another package called {metatools} in a function called `combine_supp()` (#950)
```

Comment:
Could you add brackets to the function name, i.e., `derive_vars_suppqual()`? Then it links to the documentation.

Reply:
Done, thanks.